client-server/phoebe-server

```diff
@@ -854,21 +853,18 @@ def export_params(bundleid, params):
 
     return send_file(ef.name, as_attachment=True, attachment_filename='{}_export_arrays.csv'.format(bundleid))
 
-
-
 @app.route('/bundle/<string:bundleid>', methods=['GET', 'POST'])
 @crossdomain(origin='*')
 def bundle(bundleid):
     """
     """
     j = _get_request(request)
-    clientid = j.get('clientid', None) if j is not None else None
-    client_version = j.get('client_version', None) if j is not None else None
+    #clientid = j.get('clientid', None) if j is not None else None
```

Comment:
space missing; I will no longer flag duplicates below for this or for a naked raise